 <root main_tree_to_execute = "AFL_BT" >
   <BehaviorTree ID="AFL_BT">
    <Sequence name="root_sequence">
      <WaitForPalletDetection
        name="WaitForPalletDetection"
        PalletDetectionTopic="/detected_object_front"
        PalletPose="{DetectedPalletPose}"/>
      <MoveToHeight
        name="MoveToHeight"
        PalletPose="{DetectedPalletPose}"/>
      <MoveToGoal
        name="MoveToGoal"
        PalletPose="{DetectedPalletPose}"/>
    </Sequence>
   </BehaviorTree>
 </root>
